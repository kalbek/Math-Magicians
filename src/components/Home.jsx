import React from 'react';

const Home = () => (
  <>
    <div className="flex-column-start">
      <div className="content p-5">
        <div className="sub-title">
          <h3>Welcome to our page!</h3>
        </div>
        <div className="first-paragraph">
          Welcome to Math Magicians, where learning math becomes a thrilling adventure
          for kids! Our enchanting website not only offers a handy inbuilt calculator 
          but also serves up a daily dose of motivation with inspiring quotes. 
          Join us on a journey where numbers turn into magic, and math becomes your superpower. 
          Get ready to explore, calculate, and conquer with Math Magicians!
        </div>
        <br />
        <div className="second-pareagraph">
          Math Magicians isn't just about solving equations; it's a place where young minds
          discover the wonder in numbers. Our interactive games and educational resources 
          make math not only fun but also an empowering tool for young learners.
          As they explore the world of Math Magicians, children will build a strong foundation
          for a lifetime of confidence in math, helping them unlock new opportunities 
          and embrace the world with a smile.
        </div>
      </div>
    </div>
  </>
);

export default Home;
