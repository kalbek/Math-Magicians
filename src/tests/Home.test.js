import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Home from '../components/Home';

test('It should render correctly', () => {
  const tree = render(
    <BrowserRouter>
      <Home />
    </BrowserRouter>,
  );
  expect(tree).toMatchSnapshot();
});
