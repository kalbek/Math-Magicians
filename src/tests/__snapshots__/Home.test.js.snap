// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should render correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="flex-column-start"
      >
        <div
          class="content p-5"
        >
          <div
            class="sub-title"
          >
            <h3>
              Welcome to our page!
            </h3>
          </div>
          <div
            class="first-paragraph"
          >
            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptate laudantium ullam et libero, ex, sed neque sunt voluptas, excepturi eaque tempore! Recusandae blanditiis quibusdam suscipit perferendis natus totam id numquam.
          </div>
          <br />
          <div
            class="second-pareagraph"
          >
            Lorem ipsum, dolor sit amet consectetur adipisicing elit. Similique pariatur neque ullam voluptatum culpa? Illum consequuntur odit quam velit rem perferendis autem, labore vero ipsa beatae numquam officiis quasi. Praesentium. Fuga expedita consequuntur doloribus autem temporibus architecto, saepe natus sint et distinctio facilis, quam deleniti dignissimos minus culpa totam!
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="flex-column-start"
    >
      <div
        class="content p-5"
      >
        <div
          class="sub-title"
        >
          <h3>
            Welcome to our page!
          </h3>
        </div>
        <div
          class="first-paragraph"
        >
          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Voluptate laudantium ullam et libero, ex, sed neque sunt voluptas, excepturi eaque tempore! Recusandae blanditiis quibusdam suscipit perferendis natus totam id numquam.
        </div>
        <br />
        <div
          class="second-pareagraph"
        >
          Lorem ipsum, dolor sit amet consectetur adipisicing elit. Similique pariatur neque ullam voluptatum culpa? Illum consequuntur odit quam velit rem perferendis autem, labore vero ipsa beatae numquam officiis quasi. Praesentium. Fuga expedita consequuntur doloribus autem temporibus architecto, saepe natus sint et distinctio facilis, quam deleniti dignissimos minus culpa totam!
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
